clang-tools-extra/test/clang-doc/single-source-html.cpp

```diff
@@ -0,0 +1,8 @@
+// RUN: rm -rf %t
+// RUN: mkdir %t
+// RUN: echo "" > %t/compile_flags.txt
+// RUN: cp "%s" "%t/test.cpp"
```

Comment:
Why copy this file and rename it?

Reply:
This is a pattern I've seen in other clang-doc test I thought I needed them but I think they aren't needed